01-node-tutorial/answers/03-modules.js

```diff
@@ -0,0 +1,14 @@
+const Melanie = 'Melanie'
+
+const sayHi = (name) => {
+ console.log(`Hello,my name is ${name}`)
+}
+sayHi('Melanie')
+
+const Garcia = 'Garcia'
+
```

Comment:
We also don't want to uppercase most variables,  that is for class names typically.  We capitalize global variables only if they're immutable(can't be changed) otherwise we use camelCase.